src/filesystem/windows/SDL_sysfilesystem.c

```diff
@@ -172,4 +172,22 @@ SDL_GetPrefPath(const char *org, const char *app)
 
 #endif /* SDL_FILESYSTEM_WINDOWS */
 
+#ifdef SDL_FILESYSTEM_XBOX
+#include "SDL_filesystem.h"
+char *
+SDL_GetBasePath(void)
+{
+    /* FIXME: Is this correct? */
```

Comment:
I think this may just need SDL_Unsupported(). Sometimes for consoles we try to make these work for single-player contexts but that can wait until someone has a game that would work as a test case.

Reply:
Added call to SDL_Unsupported